### DISCUSSION TITLE
Pass precalculated root element to the Dependencyparser

### DISCUSSION THREAD

**Discussion by Fxlix** [explosion/spaCy#8991] [Help: Best practices]
While processing a large amount of Sentences with german trf Model we came across unaccurate dependencies. 
i.E.: "Es wurden, so Tödter, Schäden in Höhe von gut 17 Millionen Euro ermittelt." the named entity Tödter is tagged asa root element, while its an NE not a verb. 
The part inside the commas "so NE" is a pretty common german phrasing, which refers to a statement of a person. 
With an rule based approach preceding the Parser, we already know that "wurden" is the actual root term of the Sentence. 
So is there any way to pass this knowledge to the dependencyParser pipe? 
Thanks in advance, 
Felix

**Comment by adrianeboyd** [comment]
No, sorry, the way the transition-based parser works there's no way to provide partial annotation in advance.

It doesn't look like a mistake that "wurden" is not the root in the parser output. This analysis looks like how this kind of example is annotated in the training corpus (Tiger). I'm guessing this is coming from an analysis of the sentence like "Tödter [sagt], ..." where the most suitable root given the available tokens is "Tödter" rather than "so". You can see some related examples in section 5.5, p. 73 of the guidelines: https://www.ims.uni-stuttgart.de/documents/ressourcen/korpora/tiger-corpus/annotation/tiger_scheme-syntax.pdf

We're training on a dependency conversion of the original annotation, so it's possible that some details are slightly different or that there are errors in the conversion, but it looks like the basic analysis of the sentence structure has remained the same in the training data for the provided German models.
